src/network/networkd-link.c

```diff
@@ -721,10 +721,15 @@ void link_check_ready(Link *link) {
                     !link->ipv4ll_route)
                         return;
 
-        if (link_ipv6ll_enabled(link))
+        if (link_ipv6_enabled(link)) {
+
                 if (in_addr_is_null(AF_INET6, (const union in_addr_union*) &link->ipv6ll_address) > 0)
                         return;
 
+                if (link_ipv6_accept_ra_enabled(link) && !link->ndisc_configured)
```

Comment:
I guess I got the confusion.  When we do 
```
[Match]                                                                                                                                                                                        
Name=enp0s8                                                                                                                                                                                    
                                                                                                                                                                                               
[Network]                                                                                                                                                                                      
Bridge=br0 
```
Then `link->network->bridge` gets initialized.   This indicated it's a bridge slave rather than bridge.

So the bridge will get IPv4 and IPv6 address
```
: enp0s8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc fq_codel master br0 state UNKNOWN group default qlen 1000
    link/ether 08:00:27:40:aa:db brd ff:ff:ff:ff:ff:ff
4: br0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether 96:af:f2:ea:44:34 brd ff:ff:ff:ff:ff:ff
    inet 192.168.1.106/24 brd 192.168.1.255 scope global dynamic br0
       valid_lft 86363sec preferred_lft 86363sec
    inet6 2001:db8:1:0:94af:f2ff:feea:4434/64 scope global mngtmpaddr noprefixroute dynamic 
       valid_lft 86390sec preferred_lft 14390sec
    inet6 fe80::94af:f2ff:feea:4434/64 scope link 
       valid_lft forever preferred_lft forever
```